app/Http/Controllers/MessagesController.php

```diff
@@ -92,16 +92,20 @@ public function sendMessage(Message $message)
     {
         // Get competition with activity.
         $competitionId = request('competition_id');
-        $competition = Competition::find($competitionId);
+        $competition = null;
```

Comment:
well it does get evaluated somehow. again I dont think that code is being executed, PHP is just confused because $competition was never created

![screen shot 2016-05-02 at 2 46 07 pm](https://cloud.githubusercontent.com/assets/897368/14964061/a389b01a-1074-11e6-9e6f-f6f66f716842.png)